jaxtyping/__init__.py

```diff
@@ -196,4 +197,17 @@ class PRNGKeyArray:
 del has_jax
 
 
+try:
+    import equinox
+except Exception:
+    pass
+else:
+    check_equinox_version = True  # easy-to-replace line with copybara
+    if check_equinox_version:
+        major, minor, _ = equinox.__version__
+        equinox_version = (int(major), int(minor))
+        if equinox_version < (0, 11):
+            warnings.warn("jaxtyping should be used with Equinox version >=0.11.0")
```

Comment:
I think that's the wrong place to put the version check: a particular version of Equinox is not required for the jaxtyping to work, but it is needed for tests. You probably wanted to put it into ```requirements.txt``` for the tests?

Reply:
It is required to have the import hook work with Equinox converters. I thought adding an explicit warning would help to avoid folks opening spurious bugs about it!